ArchiSteamFarm/Json/Steam.cs

```diff
@@ -477,6 +483,19 @@ internal sealed class Description {
 
 				[JsonConstructor]
 				private Description() { }
+
+				[JsonProperty(PropertyName = "tags", Required = Required.Always)]
+				private JArray Tags {
```

Comment:
`JArray` is absolutely unacceptable in types that can be statically defined. Provide full and static JSON structure definition.

Reply:
I tried, but I have no idea how to get one named value from an array. Source has a structure like this:
```"tags": [
        {
          "category": "droprate",
          "internal_name": "droprate_2",
          "localized_category_name": "Rarity",
          "localized_tag_name": "Rare"
        },
        {
          "category": "Game",
          "internal_name": "app_336420",
          "localized_category_name": "Game",
          "localized_tag_name": "Bloodsports.TV"
        },
        {
          "category": "item_class",
          "internal_name": "item_class_4",
          "localized_category_name": "Item Type",
          "localized_tag_name": "Emoticon"
        }
      ]```
I'm only interested in `"category": "droprate",`, what will be correct way to get it without getting all tags structure too?